README.md

```diff
@@ -34,128 +36,207 @@ git clone https://github.com/gnosis/dx-cli.git dx-cli
+* **Rinkeby**: 
   * [https://dutchx-rinkeby.d.exchange/api/docs/#!/markets/getMarkets](https://dutchx-rinkeby.d.exchange/api/docs/#!/markets/getMarkets)
-* **Mainnet**: The easiest way to check all token pairs that were added to the
-    DutchX is to check the API:
+* **Mainnet**: 
   * [https://dutchx.d.exchange/api/docs/#!/markets/getMarkets](https://dutchx.d.exchange/api/docs/#!/markets/getMarkets)
-* **Kovan**: Unlike `Rinkeby` and `Mainnet`, `Kovan` does't have any API
-  published. To check all available markets, you must do it at smart contract level:
+* **Kovan**: Unlike `Rinkeby` and `Mainnet`, `Kovan` doesn't have a published API. To check all available markets, you must do so on smart contract level:
   * https://dutchx.readthedocs.io/en/latest/smart-contracts_addresses.html
 
-**6. Try the CLI**
+**5. Try the CLI**
 
-Run the `help` command to get a list of all avaliable commands:
+Run the `help` command on the Terminal to get a list of all avaliable commands:
 ```bash
 # Rinkeby
-./dutchx-rinkeby help
+./dutchx-rinkeby -h
 
 # Kovan
-./dutchx-kovan help
+./dutchx-kovan -h
 
 # Mainnet
-./dutchx-mainnet help
+./dutchx-mainnet -h
 ```
 
-# Basic usage
-## Get account balances
-Get balances for all tokens of an account
+# Start Trading 
+## DutchX trading process
+Trading on the DutchX requires you to send your tokens to a smart contract, where the trading occurs. 
+The whole trading process looks as the following image:
+
+<img src="https://raw.githubusercontent.com/gnosis/dx-docs/master/source/_static/DurtchX_graphi.png" width="600" height="400">
+
+
+
+Before we start explaining each step in the image, we will use the balance command to make the process more clear. 
+All the commands used in this explanation will be for the Rinkeby Testnet. If you want to trade on Ethereum Mainnet, change `Rinkeby` to `mainnet`
+
+
 ```bash
-# Show the balances of the first account of the MNEMONIC
+# Show the balance of the account inputed in the local.config file
 ./dutchx-rinkeby balances
 
-# You can ask for the balances of any other account
-./dutchx-rinkeby balances --account=0x45345B00156Efe2a859b7E254Ab3ae0bB2eBfc0E
+# You will see the balances your balances displayed in a similar way as below:
+  INFO-cli      ACCOUNT: "Your public Ethereum address" +427ms
+  INFO-cli      BALANCE: 4.430652555999879 ETH +3ms
+  INFO-cli 
+  INFO-cli      Balances RDN (0x3615757011112560521536258c1e7325ae3b48ae): +941ms
+  INFO-cli              - Balance in DX: 222.972178357604664181 +1ms
+  INFO-cli              - Balance of user: 111.617365082280797714 +2ms
+  INFO-cli      Balances WETH (0xc778417e063141139fce010982780140aa0cd5ab): +1ms
+  INFO-cli
+  INFO-cli              - Balance in DX: 11.4032061309442109 +2ms
+  INFO-cli              - Balance of user: 0 +0ms
 ```
-## Get token list
-Get all avaliable tokens in the DX
+
+As you can see, there are two types of balance for every listed token: 
```

Comment:
For enumerations, I think is better if you use a list.

i.e.

* `Balance of user` ...
* `Balance in DX` ...